templates/notifications/_blocks.html.twig

```diff
@@ -101,22 +101,22 @@
 {% endblock magazine_ban_notification %}
 
 {% block reportlink %}
-    {% if notification.report.entry is not same as null %}
+    {% if notification.report.entry is defined and attribute(notification.report, 'entry') is not null %}
```

Comment:
Why the attribute call and not just
```suggestion
    {% if notification.report.entry is defined and notification.report.entry is not same as null %}
```

Reply:
I personally like the flexibility of attribute, but I using dot notation again.

Reply:
I was not aware of that notation and it looked funky to me :D